spec/factories/ucas_match.rb

```diff
@@ -67,7 +67,7 @@
     trait :with_multiple_acceptances do
       scheme { %w[U D] }
       application_form do
-        create(:completed_application_form, application_choices: [create(:application_choice, :with_accepted_offer)])
+        create(:completed_application_form, application_choices: [create(:application_choice, :pending_conditions)])
```

Comment:
Why has this been changed as part of this PR?

Reply:
There was some unexpected behavior when using the factories, specifically in this and the sort_applications spec, where more than the expected one application choice were being created when using some of the offer related traits. I attempted ammending the offer factory to ensure that no application_choice was created but it didn't work so it was simpler to directly use the expected application choice status as the issue only occurs in these two places.